libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-number/cell-editor-number.component.ts

```diff
@@ -23,9 +25,11 @@ import { SkyCellEditorNumberParams } from '../../types/cell-editor-number-params
 export class SkyAgGridCellEditorNumberComponent
   implements ICellEditorAngularComp
 {
-  public value: number;
```

Comment:
Removing public field is a breaking change.

Reply:
Same comment about the component not being public.